extensions/src/main/java/org/pentaho/platform/web/http/filters/PentahoWebContextFilter.java

```diff
@@ -44,20 +44,22 @@
 
 import java.io.IOException;
 import java.io.OutputStream;
+import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
-import java.util.Locale;
 import java.util.Map;
 
 /**
  * If the request is searching for a webcontext.js, it writes out the content of the webcontext.js
- *
- * @author Ramaiz Mansoor
- *
  */
 public class PentahoWebContextFilter implements Filter {
 
   public static final String WEB_CONTEXT_JS = "webcontext.js"; //$NON-NLS-1$
+
+  private static final String REQUIREJS_LOCATION = "content/common-ui/resources/web/require.js";
+  private static final String REQUIREJS_CONFIG_LOCATION = "content/common-ui/resources/web/require-cfg.js";
```

Comment:
@graimundo @dcleao Regarding `[require-cfg.js](https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/assemblies/platform-plugin/src/main/resources/resources/web/require-cfg.js)` file, it is defining `pen.require` and `pen.define`. If I remember correctly this is something that we want to remove eventually because it is legacy, but it is still being used by pentaho projects.